core/primitives/ss58_codec.cpp

```diff
@@ -28,7 +28,7 @@ namespace kagome::primitives {
       const crypto::Hasher &hasher) {
     constexpr auto PREFIX = "SS58PRE";
     auto preimage = common::Buffer{}.put(PREFIX).put(ss58_address);
-    auto checksum = hasher.blake2b_256(preimage);
+    auto checksum = hasher.blake2b_512(preimage);
```

Comment:
This diverge with SS58 documentation as far as I know, I think this requires further investigation. At  least a comment here.

Reply:
Yeah, this is something I don't understand, but two different sources - our mobile wallet dev team member Ruslan and py-substrate-interface implementation agreed that blake512 should be used for SS58. And it is still magic for me that tests work fine without modification both with 256 and 512 hashes.